doc/man3/SSL_CTX_set_options.pod

```diff
@@ -256,8 +256,12 @@ and compressed certificates will not be accepted from the peer.
 
 =item SSL_OP_NO_COMPRESSION
 
-Do not use compression even if it is supported. This option is set by default.
-To switch it off use SSL_clear_options().
+Do not use TLS record compression even if it is supported. This option is set by
+default. To switch it off use SSL_clear_options(). Note that TLS record
```

Comment:
"To switch it off" is a bit confusing..
Maybe to enable compressions use..?

Approved either way.

Reply:
I think it is ok, and since I have all approvals I will merge as is.